layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
-# Using the Vulkan GFXReconstruct Layer in Android
+# Using the GFXReconstruct Layer on Android Devices
 
 ## Before Use
 
 ### Permissions
+The GFXReconstruct layer can optionally read a configuration file
+from or write capture files to external storage.  This requires that
+the application loading the layer have the external storage permission.
 
-You will need to grant permissions to your application to write to
-system storage, even if it normally does not.
-This is due to the fact that the GFXReconstruct layer generates a
-capture file which is written to the `/sdcard` folder by default.
+The external storage permission may be requested in the application's
+manifest file.  When installing the application, it may be necessary
+to ensure that the requested permission is granted through one of
+the following actions:
 
-If you're building with Android Studio, you do this by:
+When installing the application with `adb install`:
+ * Specify the -g option: `adb install -g`
+
+When deploying from Android Studio:
  * Click on "Run" in the menu
  * Choose "Edit Configurations..."
  * In the dialog box, look for the "Install Flags:" text box
  * Enter `-g`
  * Click "Apply"
 
-If this does not work, you may still require enabling permissions for
-your application from the settings menu.
-
-**Failure to do so** will result in your application crashing during `vkCreateInstance`
-since the layer will attempt, but fail, to create the capture file.
-
-### Disabling Memory Tracking Seg-Faults
-
-The GFXReconstruct layer sometimes uses page table faults to properly
-track memory usage by an application.
-Because of this, the SIGSEGV page fault event can cause unnecessary
-breaking when stepping through a debugger while your application is
-running.
-
-To work around this, you can use the following command to stop the
-debugger from breaking on SIGSEGVs generated by the capture layer's
-memory tracking:
-
+It may also be possible to grant the external storage permission to
+an installed application through the device Settings.
+
+**Failure to enable external storage** can cause the layer to crash if
```

Comment:
"crash" here looks bad.  Is it possible to exit more gracefully, and just say "exit" instead of "crash"?

Reply:
The layer's influence over the application are limited, but there are some future changes that can be made to prevent the layer from crashing if initialization fails.  This is documenting the current behavior.